avro-serializer/src/main/java/io/confluent/kafka/serializers/AbstractKafkaAvroDeserializer.java

```diff
@@ -91,14 +93,34 @@ public DatumReader<?> load(IdentityPair<Schema, Schema> key) {
         .build(cacheLoader);
   }
 
+  protected void configure(KafkaAvroDeserializerConfig config) {
```

Comment:
FYI, I had to re-add this configure signature because other modules make use of it. e.g. `kafka-connect-avro-converter`